Awesome refactors

wolterbwambale opened the pull request:
in the project, we managed to work on the following:-

we use JavaScript classes instead of objects and arrays.
we use objects and arrays inside classes.
we created class methods to add and remove books.
we maintain all of the original functionality of the application (i.e. no bugs were introduced while refactoring).
 we Added CSS styles to the application.

wolterbwambale committed:
update js

Esmatullah12 committed:
Update the main.js file

refactor the code main.js
- Use JavaScript classes instead of objects and arrays. 
- Create class methods to add and remove books.

wolterbwambale committed:
add book.css file

wolterbwambale committed:
update css file

omarsalem7 commented on main.js:
```diff
@@ -1,59 +1,64 @@
-  // Reset the form
-  addBookForm.reset();
-});
+/* eslint-disable */ 
```
- [ ] Kindly consider avoiding disabling the linter because it helps keep code tidy and easier to understand and use throughout the project for anyone who needs to look at the project code in the future

omarsalem7 reviewed (changes requested):
Hi @wolterbwambale, @Esmatullah12 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

![you-are-near](https://media.giphy.com/media/j7vckgz15cAYmxatzB/giphy.gif)

## Highlights
- Good use of JavaScript classes.✔️
- The Pull Request has a proper title and description. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Esmatullah12 committed:
Update main.js file

-remove the book class.
-modify the addbook method.
-modify the removeBook method.
-modify the createBookElement book

wolterbwambale committed:
fix required changes

wolterbwambale commented:
@omarsalem7 hello we have fixed the required changes.

victorokoroji reviewed (approved):
Hi @wolterbwambale  

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Requested change implemented ✅ 

<img src="https://media3.giphy.com/media/xT0xezQGU5xCDJuCPe/giphy.gif"/>

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
